### PR TITLE
Theming for Skin (Main skin)

### PR DESCRIPTION
**Pull request by Karolis2011**
This is another version of #6926
And this branch if based on #7001 

This is [how this looks.](https://cdn.discordapp.com/attachments/131489773268238336/625068068799905793/1569098187_kFXI.mp4)

### CODE REVIEW

**Comment by alsoandanswer** [comment]
oh fuck

**Comment by Karolis2011** [comment]
_From now on I won't allow edits from maintainers, because people don't know what they should press or what they shouldn't_

File: SQL/migrate/V043__Skin_Themes.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+--
+-- Adds HTML style value to the player preferences table.
+--
+
+ALTER TABLE `ss13_player_preferences`
+	ADD `skin_theme` VARCHAR(32) DEFAULT 'Ligth';
```
Comment (NonQueueingMatt) [inline SQL/migrate/V043__Skin_Themes.sql]:
```suggestion
	ADD `skin_theme` VARCHAR(32) DEFAULT 'Light';
```

File: code/controllers/subsystems/theming.dm [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+			"type" = THEME_TYPE_LIGHT
+		),
+		"Basic Dark" = list(
+			"name" = "Basic Dark",
+			"class" = "theme-basic-dark",
+			"type" = THEME_TYPE_DARK
+		)
+	)
+
+	var/skin_files = list("Ligth" = "interface/skin.txt", "Dark" = "interface/dark.txt")
```
Comment (NonQueueingMatt) [inline code/controllers/subsystems/theming.dm]:
```suggestion
	var/skin_files = list("Light" = "interface/skin.txt", "Dark" = "interface/dark.txt")
```

File: code/modules/client/preferences.dm [hunk 3]
```diff
@@ -27,6 +27,7 @@ datum/preferences
 	var/UI_style_color = "#ffffff"
 	var/UI_style_alpha = 255
 	var/html_UI_style = "Nano"
+	var/skin_theme = "Ligth"
```
Comment (NonQueueingMatt) [inline code/modules/client/preferences.dm]:
```suggestion
	var/skin_theme = "Light"
```

**Comment by alsoandanswer** [comment]
What anout that black text in the video (the one that says please choose your character in the sidebar)

Shouldm't it be changing to white text

**Comment by Karolis2011** [comment]
@alsoandanswer I would like to point out that chat being broken is intended consequence.

**Pull request merged**